server/utils/pagination.test.ts

```diff
@@ -13,8 +13,8 @@ describe('pagination', () => {
     expect(pagination(1, 2, '?a=b&')).toEqual<Pagination>({
       next: { href: '?a=b&page=2' },
       items: [
-        { number: 1, href: '?a=b&page=1', current: true },
-        { number: 2, href: '?a=b&page=2' },
+        { text: '1', href: '?a=b&page=1', selected: true },
```

Comment:
I think it would be useful to include a sentence about why you're opting to use MoJ Pagination over GDS Pagination in the commit description